draft-ietf-scitt-software-use-cases.md

```diff
@@ -56,8 +56,8 @@ These use cases aim to identify software supply chain problems that the industry
 # Introduction {#intro}
 
 Modern software applications are an intricate mix of first-party and third-party code, development practices and tools, deployment methods and infrastructure, and interfaces and protocols.
-The software supply chain comprises all elements associated with an application's design, development, build, integration, deployment, and maintenance throughout its entire lifecycle.
-The complexity of software coupled with a lack of lifecycle visibility increases the risks associated with system attack surface and the number of cyber threats capable of harmful impacts, such as exfiltration of data, disruption of operations, and loss of reputation, intellectual property, and financial assets.
+The software supply chain comprises all elements associated with a system's design, development, build, integration, deployment, and maintenance throughout its entire lifecycle.
```

Comment:
Minor, non-blocking: I am going to note that in reviewing drafts and engaging in the working group, it is clear that an application will want to be assessed as its component parts and where it is deployed as a component of a larger system, so I think it is good to adjust this accordingly. Thoughts?